### PR TITLE
Tweak: Adjustments to workflow hints for Image Optimizer [ED-13844]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: core/utils/hints.php [hunk 1]
```diff
@@ -130,9 +130,9 @@ public static function get_notice_template( array $notice, bool $return = false
 		}
 
 		if ( $notice_settings['dismissible'] ) {
-			$dismissible = '<button class="elementor-control-notice-dismiss" data-event="' . $notice_settings['dismissible'] . '">
+			$dismissible = '<button class="elementor-control-notice-dismiss tooltip-target" data-event="' . $notice_settings['dismissible'] . '" data-tooltip="' . esc_attr__( "Dismiss and don't show again.", 'elementor' ) . '">
 				<i class="eicon eicon-close" aria-hidden="true"></i>
-				<span class="elementor-screen-only">' . __( 'Dismiss this notice.', 'elementor' ) . '</span>
+				<span class="elementor-screen-only">' . __( "Dismiss and don't show again.", 'elementor' ) . '</span>
```
Comment (arielk) [inline core/utils/hints.php:135]: Don't Show Again (remove dismiss)

File: includes/controls/notice.php [hunk 2]
```diff
@@ -80,9 +80,9 @@ public function content_template() {
 				<# } #>
 			</div>
 			<# if ( data.dismissible ) { #>
-			<button class="elementor-control-notice-dismiss">
+			<button class="elementor-control-notice-dismiss tooltip-target" data-tooltip="<?php echo esc_html__( 'Dismiss and don’t show again.', 'elementor' ); ?>">
 				<i class="eicon eicon-close" aria-hidden="true"></i>
-				<span class="elementor-screen-only"><?php echo esc_html__( 'Dismiss this notice.', 'elementor' ); ?></span>
+				<span class="elementor-screen-only"><?php echo esc_html__( 'Dismiss and don’t show again.', 'elementor' ); ?></span>
```
Comment (arielk) [inline includes/controls/notice.php:85]: Don't Show Again (remove dismiss)

File: includes/controls/notice.php [hunk 3]
```diff
@@ -80,9 +80,9 @@ public function content_template() {
 				<# } #>
 			</div>
 			<# if ( data.dismissible ) { #>
-			<button class="elementor-control-notice-dismiss">
+			<button class="elementor-control-notice-dismiss tooltip-target" data-tooltip="<?php echo esc_html__( 'Dismiss and don’t show again.', 'elementor' ); ?>">
```
Comment (arielk) [inline includes/controls/notice.php:83]: Don't Show Again (remove dismiss)

File: includes/controls/media.php [hunk 4]
```diff
@@ -253,7 +253,7 @@ public function content_template() {
 						<?php
```
Comment (arielk) [inline includes/controls/media.php:253]: Did you added close button for the SEO notice?

File: includes/controls/media.php [hunk 5]
```diff
@@ -263,21 +263,33 @@ public function content_template() {
...
+									);
+								} else {
+									$content = sprintf(
+										__( 'This image is large and may slow things down. <a href="%s">Install Image Optimizer</a> to reduce size without losing quality.', 'elementor' ),
+										Hints::get_plugin_action_url( 'image-optimization' )
+									);
+								}
+							} else {
+								$content = sprintf(
+									'%1$s <a href="%2$s">%3$s</a>',
```
Comment (arielk) [inline includes/controls/media.php:280]: Add target blank

Comment (rami-elementor) [reply includes/controls/media.php]: Done

**Review by arielk — APPROVED** [review]

**Pull request merged**